libvast/src/system/configuration.cpp

```diff
@@ -103,16 +103,16 @@ caf::error configuration::parse(int argc, char** argv) {
     std::error_code err{};
     const auto exists_conf_yaml = std::filesystem::exists(conf_yaml, err);
     const auto exists_conf_yml = std::filesystem::exists(conf_yml, err);
-    if (err)
-      return caf::make_error(ec::filesystem_error,
-                             fmt::format("failed to check if vast yaml config "
-                                         "files existed from directory {}: {}",
-                                         dir, err.message()));
+    if (err) {
+      VAST_WARN("failed to check if vast.yaml file exists in {}: {}", dir,
+                err.message());
```

Comment:
I know I'm late to the party, but I'm missing a bit of user hand-holding in the communication: what should the user do now? Is everything still okay?

Reply:
The error message says what's wrong with quite some detail, so I think this suffices.